environments/pull-environment.sh

```diff
@@ -2,8 +2,12 @@
 set -e
 IFS='|'
 
+# Get args
+dir=$1
+region=$2
+
 # In development, AWS_PROFILE should be set. In CI, it's not.
-[ "$AWS_PROFILE" ] && useProfile="true" || useProfile="false";
+[ "$AWS_PROFILE" ] && useProfile="true" || useProfile="false"
```

Comment:
Just a style adjustment: ending line with `;` is [superfluous](https://stackoverflow.com/questions/7507038/semicolons-superfluous-at-the-end-of-a-line-in-shell-scripts) in bash.